migrate/dual-write-and-backfill/dual-write-from-timescaledb.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Migrate from TimescaleDB using dual-write and backfill
+excerpt: Migrate from a TimescaleDB database using the low-downtime dual-write and backfill method
+products: [cloud]
+keywords: [migration, low-downtime]
+tags: [migration, logical backup]
+---
+
+import GettingHelp from "versionContent/_partials/_migrate_dual_write_backfill_getting_help.mdx";
+import StepOne from "versionContent/_partials/_migrate_dual_write_step1.mdx";
+import StepTwo from "versionContent/_partials/_migrate_dual_write_step2.mdx";
+import StepFour from "versionContent/_partials/_migrate_dual_write_step4.mdx";
+import StepFive from "versionContent/_partials/_migrate_dual_write_step5.mdx";
+import SetupSourceTarget from "versionContent/_partials/_migrate_set_up_source_and_target.mdx";
+import ValidateProductionLoad from "versionContent/_partials/_migrate_dual_write_validate_production_load.mdx";
+import SwitchProductionWorkload from "versionContent/_partials/_migrate_dual_write_switch_production_workload.mdx";
+import SourceTargetNote from "versionContent/_partials/_migrate_source_target_note.mdx";
+import TimescaleDBVersion from "versionContent/_partials/_migrate_from_timescaledb_version.mdx";
+
+# Dual-write and backfill from TimescaleDB database
+
+This document provides detailed step-by-step instructions to migrate data using
+the [dual-write and backfill][dual-write-and-backfill] migration method from a
+source database which is using TimescaleDB to Timescale.
+
+<SourceTargetNote />
+
+In detail, the migration process consists of the following steps:
+1. Set up a target database instance in Timescale.
+1. Modify the application to write to a secondary database.
+1. Migrate schema and relational data from source to target.
+1. Start the application in dual-write mode.
+1. Determine the completion point `T`.
+1. Backfill time-series data from source to target.
+1. Enable background jobs (policies) in the target database.
+1. Validate that all data is present in target database.
+1. Validate that target database can handle production load.
+1. Switch application to treat target database as primary (potentially
+   continuing to write into source database, as a backup).
+
+<GettingHelp />
+
+<StepOne />
+
+<StepTwo />
+
+## 3. Set up schema and migrate relational data to target database
+
+This section leverages `pg_dumpall` and `pg_dump` to migrate the roles and
+relational schema that you are using in the source database to the target
+database.
+
+<Highlight type="important">
+The PostgresSQL versions of the source and target databases can be of different
+versions, as long as the target version is greater than that of the source.
+
+The version of TimescaleDB used in both databases must be exactly the same.
+</Highlight>
+
+### 3a. Dump the database roles from the source database
```

Comment:
I think this section is same as what is mentioned in `from postgres` approach. You might wanna abstract this inside `_partials`

Reply:
👍  done